airbyte-webapp-e2e-tests/cypress/integration/connection.spec.js

```diff
@@ -1,12 +1,16 @@
 describe("Connection main actions", () => {
+  beforeEach(() => {
+    cy.initialSetupCompleted();
+  });
+
   it("Create new connection", () => {
     cy.createTestConnection("Test connection source cypress", "Test destination cypress");
 
     cy.get("div").contains("Test connection source cypress").should("exist");
     cy.get("div").contains("Test destination cypress").should("exist");
   });
 
-  it.only("Update connection", () => {
```

Comment:
:information_source: This was (I guess) accidentally merged from https://github.com/airbytehq/airbyte/pull/9025/files